.github/workflows/release.yml

```diff
@@ -113,20 +113,20 @@ jobs:
       - name: Run unit tests
         if: ${{ env.PACKAGES_CHANGES == 'true' || github.ref == 'refs/heads/main' }}
         run: pnpm test
-      - name: Get installed Playwright version
-        id: playwright-version
-        run: echo "PLAYWRIGHT_VERSION=$(pnpx playwright -V | awk '{ print $2 }')" >> $GITHUB_ENV
-      - name: Cache playwright binaries
-        uses: actions/cache@v4
-        id: playwright-cache
-        with:
-          path: |
-            ~/.cache/ms-playwright
-          key: ${{ runner.os }}-playwright-${{ env.PLAYWRIGHT_VERSION }}
       - name: Install Playwright
-        if: steps.playwright-cache.outputs.cache-hit != 'true' && github.repository == 'nextauthjs/next-auth'
+        if: github.repository == 'nextauthjs/next-auth'
         run: |
           pnpm exec playwright install --with-deps chromium
+      - name: Pass environment variables to Playwright
```

Comment:
So I think this might be the issue.. These env vars are in github secrets, which are accessed differently. Let's try adding the following to after like 78, so they apply to this whole `test` job. Something like:

```yml
  env:
    AUTH_SECRET: ${{ secrets.AUTH_SECRET }}
    TEST_KEYCLOAK_USERNAME: ${{ secrets.TEST_KEYCLOAK_USERNAME }}
    TEST_KEYCLOAK_PASSWORD: ${{ secrets.TEST_KEYCLOAK_PASSWORD }}
    AUTH_KEYCLOAK_ID: ${{ secrets.AUTH_KEYCLOAK_ID }}
    AUTH_KEYCLOAK_SECRET: ${{ secrets.AUTH_KEYCLOAK_SECRET }}
    AUTH_KEYCLOAK_ISSUER: ${{ secrets.AUTH_KEYCLOAK_ISSUER }}
    AUTH_TRUST_HOST: ${{ secrets.AUTH_TRUST_HOST }}
```